WalletWasabi/Services/WasabiSynchronizer.cs

```diff
@@ -23,7 +24,7 @@
 
 namespace WalletWasabi.Services
 {
-	public class WasabiSynchronizer : NotifyPropertyChangedBase
+	public class WasabiSynchronizer : NotifyPropertyChangedBase, IFeeProvider
```

Comment:
This could be a `PeriodicRunner` too.

Reply:
Good idea, but I would prefer not touching it here, because it has a set of custom behaviors.